services/frontend-service/Makefile

```diff
@@ -14,8 +14,10 @@
 #along with kuberpult.  If not, see <http://www.gnu.org/licenses/>.
 
 #Copyright 2021 freiheit.com
-MAKEFLAGS += --no-builtin-rules
 
+include ../../Makefile.variables
+
+MAKEFLAGS += --no-builtin-rules
```

Comment:
Why is this needed?

Reply:
makeflags already existed.
the makefile variable was added to get IMAGE_REGISTRY variable